app/views/candidate_interface/start_page/create_account_or_sign_in.html.erb

```diff
@@ -1,16 +1,18 @@
-<% content_for :title, title_with_error_prefix(t('page_titles.create_account_or_sign_in'), @create_account_or_sign_in_form.errors.any?) %>
+<% title = FeatureFlag.active?(:one_login_candidate_sign_in) ? t('page_titles.create_a_gov_uk_one_login_or_sign_in') : t('page_titles.create_account_or_sign_in') %>
+<% content_for :title, title_with_error_prefix(title, @create_account_or_sign_in_form.errors.any?) %>
```

Comment:
Can we extract this logic out of the view and into a helper method? 

I find it difficult to read with the combination of translations and the use of a ternary operator.